### PR TITLE
Move The Radius Audit Log in it's own section

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Move The Radius Audit Log in it's own section 
## Impacts

Admin GUI

Enhancements
++++++++++++
- Moved radius audit log in it's own section
- Saved searches for the Radius Audit Log

Bug Fixes
+++++++++
## Delete branch after merge

NO

### CODE REVIEW

**Comment by fdurand** [comment]
Still have the old way to reach the radius audit log

**Comment by fdurand** [comment]
Rename Radius Log to Audit in the top of the admin gui (we will probably add more Audit stuff)

**Comment by extrafu** [comment]
"Audit" for French, "Auditing" for English please.

File: html/pfappserver/root/admin/radiuslog.tt [hunk 1]
```diff
@@ -0,0 +1,55 @@
+[%  jsFiles = [
```
Comment (julsemaan) [inline html/pfappserver/root/admin/radiuslog.tt]: Shouldn't this file be named auditing.tt even if it only contains the RADIUS audit log now.

My understanding is that more reports will join that section shortly (admin auditing) and will share the page as a base.

File: html/pfappserver/lib/pfappserver/Controller/SavedSearch/RadiusLog.pm [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package pfappserver::Controller::SavedSearch::RadiusLog;
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Controller/SavedSearch/RadiusLog.pm:1]: Same comment about the naming RadiusLog vs Auditing

File: html/pfappserver/lib/pfappserver/Model/SavedSearch/RadiusLog.pm [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package pfappserver::Model::SavedSearch::RadiusLog;
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Model/SavedSearch/RadiusLog.pm:1]: Same comment about the naming RadiusLog vs Auditing

**Comment by jrouzierinverse** [comment]
IMO we should just rename the heading to Auditing and create the structure to have more than one type of auditing reports.
Such as 
- authlog
- json auditing log.

IMO the models, controller, and templates for the RadiusLog should stay separate from the main auditing section to make it easier to add and remove audit reports.

With that said it may make sense to have a separate namespace for the auditing models and controllers.
Such as 
- Model::SavedSearch::Auditing::RadiusLog
- Model::Auditing::RadiusLog
- Controller::Auditing::RadiusLog
- Controller::SavedSearch::Auditing::RadiusLog

Let me know what you think.

**Comment by julsemaan** [comment]
Yes that would be perfect and will be easy to extend once we bring the other auditing pieces in the admin

**Comment by jrouzierinverse** [comment]
Please review

File: html/pfappserver/lib/pfappserver/Model/Auditing/RadiusLog.pm [hunk 4]
```diff
@@ -111,14 +111,11 @@ sub _add_date_range {
     my $start = $params->{start};
     my $end = $params->{end};
     return unless defined $start;
-    if (defined $start && defined $end) {
-        my $start_date = "$start->{date} $start->{time}";
-        my $end_date = "$end->{date} $end->{time}";
+    my $start_date = "$start->{date} " . $start->{time} // "00:00";
+    if (defined $end) {
+        my $end_date = "$end->{date} " . $end->{time} // "23:59";
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Model/Auditing/RadiusLog.pm]: I know i'm being picky but shouldn't that be 23:59:59 ?

File: html/pfappserver/lib/pfappserver/PacketFence/Controller/SavedSearch/RadiusLog.pm [hunk 5]
```diff
@@ -0,0 +1,78 @@
+package pfappserver::PacketFence::Controller::SavedSearch::RadiusLog;
+
+=head1 NAME
+
+pfappserver::PacketFence::Controller::SavedSearch::RadiusLog add documentation
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/PacketFence/Controller/SavedSearch/RadiusLog.pm]: add it

File: html/pfappserver/lib/pfappserver/PacketFence/Controller/Admin.pm [hunk 6]
```diff
@@ -252,6 +252,20 @@ sub reports :Chained('object') :PathPart('reports') :Args(0) :AdminRole('REPORTS
     $c->forward('Controller::Graph', 'reports');
 }
 
+=head2 auditing
+
+=cut
+
+sub auditing :Chained('object') :PathPart('auditing') :Args(0) :AdminRole('RADIUS_LOG_READ') {
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/PacketFence/Controller/Admin.pm]: Shouldn't that be AUDITING_READ ?

File: html/pfappserver/lib/pfappserver/Model/Auditing/RadiusLog.pm (hunk 4 above)
Comment (jrouzierinverse) [inline html/pfappserver/lib/pfappserver/Model/Auditing/RadiusLog.pm]: Yes you are correct it should be 23:59:59.

**Pull request merged**